Modelica/Blocks/package.mo

```diff
@@ -927,6 +927,88 @@ Note, that
+    Logical.Or or2
+      annotation (Placement(transformation(extent={{10,-30},{30,-10}})));
+    Logical.MultiOr multiOr(nu=3)
+      annotation (Placement(transformation(extent={{10,-60},{30,-40}})));
+    Logical.Not notA
+      annotation (Placement(transformation(extent={{50,70},{70,90}})));
+    Logical.MultiNand multiNand(nu=3)
+      annotation (Placement(transformation(extent={{50,20},{70,40}})));
+    Logical.Not notB
+      annotation (Placement(transformation(extent={{50,-30},{70,-10}})));
+    Logical.MultiNor multiNor(nu=3)
+      annotation (Placement(transformation(extent={{50,-80},{70,-60}})));
+  equation
+    connect(booleanPulseA[1].y, and1.u1) annotation (Line(points={{-59,50},{-50,
+            50},{-50,80},{-30,80}},
+                                color={255,0,255}));
+    connect(booleanPulseA[2].y, and1.u2) annotation (Line(points={{-59,50},{-40,
+            50},{-40,72},{-30,72}},
+                                color={255,0,255}));
+    connect(and1.y, and2.u1)
+      annotation (Line(points={{-7,80},{8,80}}, color={255,0,255}));
+    connect(booleanPulseA[3].y, and2.u2) annotation (Line(points={{-59,50},{0,50},
+            {0,72},{8,72}},color={255,0,255}));
+    connect(booleanPulseB[1].y, or1.u1) annotation (Line(points={{-59,-50},{-50,
+            -50},{-50,-20},{-32,-20}}, color={255,0,255}));
+    connect(booleanPulseB[2].y, or1.u2) annotation (Line(points={{-59,-50},{-40,
+            -50},{-40,-28},{-32,-28}}, color={255,0,255}));
+    connect(or1.y, or2.u1)
+      annotation (Line(points={{-9,-20},{8,-20}}, color={255,0,255}));
+    connect(booleanPulseB[3].y, or2.u2) annotation (Line(points={{-59,-50},{0,
+            -50},{0,-28},{8,-28}}, color={255,0,255}));
+    connect(booleanPulseA.y, multiAnd.u)
+      annotation (Line(points={{-59,50},{10,50}}, color={255,0,255}));
+    connect(booleanPulseB.y, multiOr.u)
+      annotation (Line(points={{-59,-50},{10,-50}}, color={255,0,255}));
+    connect(and2.y, notA.u)
+      annotation (Line(points={{31,80},{48,80}}, color={255,0,255}));
+    connect(booleanPulseA.y, multiNand.u) annotation (Line(points={{-59,50},{0,
+            50},{0,30},{50,30}}, color={255,0,255}));
+    connect(or2.y, notB.u)
+      annotation (Line(points={{31,-20},{48,-20}}, color={255,0,255}));
+    connect(booleanPulseB.y, multiNor.u) annotation (Line(points={{-59,-50},{0,
+            -50},{0,-70},{50,-70}}, color={255,0,255}));
+    annotation (Icon(coordinateSystem(preserveAspectRatio=false)), Diagram(
+          coordinateSystem(preserveAspectRatio=false)),
+      experiment(StopTime=1),
+      Documentation(info="<html>
+<p>
+Compare and / or / nand / nor for (only) 3 Boolean input signals, 
+logical operations performed pairwise and as vector (or arbitrary length). 
+Compare:
+</p>
+<ul>
+<li>and2.y vs. multiAnd.y</li>
+<li>notA.y vs. multiNand.y</li>
+<li>or2.y  vs. multiOr.y</li>
+<li>notB.y vs. multiNor.y</li>
```

Comment:
I am not sure of compare `a` vs `b` is the correct terminology. This is how I understand it:

1. Compare `a` vs `b` means to me that `a` is on the vertical axis, `b` is on the horizontal axis
2. Compare `a` with `b` means to me that `a` and `b` are plotted as a function of time (in the same graph)
3. Compare `a` and `b` means to me the same as 2.

I am kindly asking for a native speaker to comment on this